lib/Resque.php

```diff
@@ -132,72 +138,9 @@ public static function pop($queue)
 	}
 
 	/**
-	 * Remove items of the specified queue
-	 *
-	 * @param string $queue The name of the queue to fetch an item from.
-	 * @param array $items
-	 * @return integer number of deleted items
-	 */
-	public static function dequeue($queue, $items = Array())
-	{
-	    if(count($items) > 0) {
-		return self::removeItems($queue, $items);
-	    } else {
-		return self::removeList($queue);
-	    }
-	}
-
-	/**
-	 * Remove specified queue
-	 *
-	 * @param string $queue The name of the queue to remove.
-	 * @return integer Number of deleted items
-	 */
-	public static function removeQueue($queue)
-	{
-	    $num = self::removeList($queue);
-	    self::redis()->srem('queues', $queue);
-	    return $num;
-	}
```

Comment:
We've grown rather fond of being able to cancel jobs after they've been queued.  Can't remove this feature.